src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+/**
+ *
+ */
```

Comment:
I think there is a missing javadocs header here? 😄

Reply:
Agreed, seems like it's walked off somewhere. 😄